packages/stark-ui/src/modules/table/entities/column-properties.intf.ts

```diff
@@ -86,4 +86,16 @@ export interface StarkTableColumnProperties {
 	 * Priority of the column. Default: PERSIST
 	 */
 	priority?: StarkTableColumnPriority;
+
+	/**
+	 * Whether the column should be shown on the right side of the table (stickyEnd)
```

Comment:
Also I think this should be: "Whether the column should be **fixed** to the right side of the table (stickyEnd)".